### ISSUE TITLE
Cannot import magento lib variables less from module scope

### ISSUE THREAD

**Issue by mrtuvn** [magento/magento2#36420]
### Preconditions and environment

Any version

### Steps to reproduce

Go to app/code/Vendor/Module/web/css

Create file styles.less for example

Inside import less code 
Full path
app/code/Vendor/Module/web/css/styles.less

```less
@import 'source/lib/_variables.less';
@import 'source/lib/_responsive.less';
```
https://github.com/magento/magento2/blob/2.4-develop/lib/web/css/source/lib/_variables.less
https://github.com/magento/magento2/blob/2.4-develop/lib/web/css/source/lib/_responsive.less
### Expected result

Variables in file lib/web/css/source/lib/_variables.less should be imported and can access inside file styles.less from module area

### Actual result

Variables/Tokens not included
Got error while compile source

Unable to get content for 'frontend/Magento/blank/en_US/Vendor_Module/css/source/lib/_variables.less'

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by engcom-November** [comment]
Thanks for the report and collaboration!
We have tried to reproduce this issue in Magento 2.4-develop branch by making a custom module but the issue is not reproducible. We are attaching that module below for reference:
[Vendor.zip](https://github.com/magento/magento2/files/10422910/Vendor.zip)

We have followed the below steps:

    Created custom Module 
    Created styles.less in custom Module in app/code/Vendor/Module/web/css
    Imported less code into module can see in styles.less
    Ran compile command  php bin/magento setup:di:compile 

Kindly recheck the issue on Magento 2.4-develop instance and elaborate missing steps if any if the issue is still reproducible.
Thank you.
